src/operations/extrusions/slice/create.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Creates a new empty slice
+ *
+ * @returns {slice} a new slice
+ */
+const create = (edges) => {
```

Comment:
hmm so a slice is a new structure, that is similar to a geom 2 ? or are internal holes not possible ?

Reply:
Yeah. That's why this new data structure was created. The edges are ordered, and therefore any number of holes are possible.

Reply:
I thought geom2 was ordered as well ?

Reply:
Correct. geom2 has 2D edges, called sides. The points are ordered due to the sides.